engine/source/runtime/function/render/render_pipeline.cpp

```diff
@@ -192,8 +194,8 @@ namespace Pilot
         fxaa_pass.updateAfterFramebufferRecreate(
             main_camera_pass.getFramebufferImageViews()[_main_camera_pass_post_process_buffer_odd]);
         combine_ui_pass.updateAfterFramebufferRecreate(
-            main_camera_pass.getFramebufferImageViews()[_main_camera_pass_backup_buffer_even],
-            main_camera_pass.getFramebufferImageViews()[_main_camera_pass_backup_buffer_odd]);
+            main_camera_pass.getFramebufferImageViews()[_main_camera_pass_backup_buffer_odd],
```

Comment:
why change the order?

Reply:
the `_main_camera_pass_backup_buffer_odd` VkImageView is used as the `scene_input_attachment` now no matter enable or disable FXAA pass. and the `_main_camera_pass_backup_buffer_even` is used as the `ui_input_attachment`. it doesn't make any difference if not changed the order. for easy understanding, i think the parameter should follow the definition `void CombineUIPass::updateAfterFramebufferRecreate(VkImageView scene_input_attachment, VkImageView ui_input_attachment)`.